en/components/calendar.md

```diff
@@ -234,6 +234,101 @@ Having implemented this conditional templating and date parsing we should get co
     <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="calendar-sample-4-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
 </div>
 
+### Disabled dates
+This section demonstrates the usage of `disabledDates` functionallity. Different `single dates` or `range` elements could be added to Array, and passed to the `disabledDates` descriptor.
+
+```typescript
+this.calendar.disabledDates = [{ type: DateRangeType.Between, dateRange: [
+    new Date(2018, 8, 2),
+    new Date(2018, 8, 8)
+])];
+```
+
+The `DateRangeType` is used to specify the range that is going to be disabled. For example, `DateRangeType.Between` will disable the dates between two specific dates in Array. Code snipped above.
```

Comment:
Should be `snippet` instead of `snipped`.

Reply:
Done